package-lock.json

```diff
@@ -59,6 +59,9 @@
         "stylelint-prettier": "^5.0.2",
         "typescript": "^5.6.2",
         "vitest": "^2.0.5"
+      },
+      "engines": {
+        "node": ">=20.X"
```

Comment:
Shouldn't it be a lowercase "X" like in `package.json`?
![image](https://github.com/user-attachments/assets/744137a0-1026-4e5f-aa9e-a6d4e8bf7e32)

Reply:
fixed